tests/HTTP/SapiTest.php

```diff
@@ -117,4 +117,49 @@ function testSend() {
 
     }
 
+    /**
+     * @runInSeparateProcess
+     */
+    function testSendLimitedByContentLengthString() {
+
+        $response = new Response(200);
+
+        $response->addHeader('Content-Length', 19);
+        $response->setBody('Send this sentence. Ignore this one.');
+
+        ob_start();
+
+        Sapi::sendResponse($response);
+
+        $result = ob_get_clean();
+        header_remove();
```

Comment:
It removes all headers, including vendors (`X-…`). I wonder if some test engine uses HTTP headers for some stuff. But nevermind, we will see. Since PHPUnit does not execute tests in parallel, this is useful.